deposit/src/main/resources/fcrepo-clients-context.xml

```diff
@@ -48,8 +48,15 @@
 		<property name="aclService" ref="aclService" />
 	</bean>
 	
-	<bean id="repository" class="edu.unc.lib.dl.fcrepo4.Repository">
+	<bean id="repositoryObjectCacheLoader" class="edu.unc.lib.dl.fcrepo4.RepositoryObjectCacheLoader">
 		<property name="client" ref="fcrepoClient" />
+		<property name="repository" ref="repository" />
```

Comment:
the order in which beans are initialized gets worked out by spring as long as there aren't any impossible situations (such as two constructors referring to eachother's objects.  which is why injecting the dependencies as setters instead of as constructor parameters is often safer)